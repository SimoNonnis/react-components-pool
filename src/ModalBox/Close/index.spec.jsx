/* eslint-env jest */
import React from 'react';

import ModalClose from './';

describe('Test <ModalClose />', () => {
  const mockOnClick = jest.fn();

  it('should render ModalClose correctly with required props', () => {
    const wrapper = shallow(<ModalClose onClick={mockOnClick} />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render ModalClose and call onClick when clicked', () => {
    const wrapper = shallow(<ModalClose onClick={mockOnClick} />);
    wrapper.simulate('click');
    expect(mockOnClick).toHaveBeenCalled();
  });
});
