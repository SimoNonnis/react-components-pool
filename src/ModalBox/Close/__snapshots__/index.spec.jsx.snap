// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <ModalClose /> should render ModalClose correctly with required props 1`] = `
<CloseModal
  onClick={[MockFunction]}
>
  Ã—
</CloseModal>
`;
